### PR TITLE
Post date, title, and content vertically misaligned #162

### PR DESCRIPTION
**Pull request by akshayvinchurkar**
This issue is small but i fixed it with this small tweak

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1381,6 +1381,7 @@ body.page:not(.twentyseventeen-front-page) .entry-title {
 	text-decoration: none;
 	-webkit-transition: color 0.2s;
 	transition: color 0.2s;
+	margin-left:-2px;
```
Comment (laurelfulford) [inline style.css]: Minor formatting change - please add a space after the colon.

File: style.css [hunk 2]
```diff
@@ -3338,4 +3339,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```
Comment (laurelfulford) [inline style.css:3342]: Please don't remove the line break at the end of the file - thanks!

**Comment by laurelfulford** [comment]
Thanks for this PR, @akshayvinchurkar!

I added a couple notes for changes in the code. Could you also update the rtl.css styles to match, so they remove this negative left margin and add a negative right margin instead?

Let me know if you have any questions at all about this - thanks!

File: style.css (hunk 1 above)
Comment (akshayvinchurkar) [reply style.css]: i fix this

File: style.css [hunk 3]
```diff
@@ -3328,14 +3329,15 @@ article.panel-placeholder {
 }
 
 @media screen and ( min-width: 79em ) {
+	
 	.has-sidebar .entry-content blockquote.alignleft {
 		margin-left: -20%;
 	}
-
+
```
Comment (ntwb) [inline style.css:3335]: Remove this tabbed whitespace

File: style.css [hunk 4]
```diff
@@ -3328,14 +3329,15 @@ article.panel-placeholder {
...
+	
 	.blog:not(.has-sidebar) .entry-content blockquote.alignright,
 	.archive:not(.has-sidebar) .entry-content blockquote.alignright,
 	.page-two-column .entry-content blockquote.alignright,
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
-	}
-}
\ No newline at end of file
+	}
```
Comment (ntwb) [inline style.css:3341]: Also remove this tabbed whitespace
Edit: This one is a little harder to see in the code review, there is a tab after the closing brace that shouldn't be there

**Review by ntwb — CHANGES_REQUESTED** [review]
A couple of issues of whitespace added where it should not be

File: style.css [hunk 5]
```diff
@@ -3330,8 +3330,8 @@ article.panel-placeholder {
 
 @media screen and ( min-width: 79em ) {
 	.has-sidebar .entry-content blockquote.alignleft {
-		margin-left: -20%;
-	}
+	        margin-left: -20%;
+        }
```
Comment (ntwb) [inline style.css:3335]: Tabs not spaces please

File: style.css [hunk 6]
```diff
@@ -3330,8 +3330,8 @@ article.panel-placeholder {
 
 @media screen and ( min-width: 79em ) {
 	.has-sidebar .entry-content blockquote.alignleft {
-		margin-left: -20%;
-	}
+	        margin-left: -20%;
```
Comment (ntwb) [inline style.css:3333]: Tabs not spaces please

**Review by ntwb — CHANGES_REQUESTED** [review]
Lines 3333 & 3334 are indented with spaces, they should be indented with tabs please

**Comment by davidakennedy** [comment]
Adjusted and merged in #333.

**Pull request closed without merging**